<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QMainWindow" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Параметры:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboParameters"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>m =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinM">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>W0 =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="spinW0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>z1 =</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinZ1">
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>E =</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="spinE">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>z2 =</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinZ2">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>x2 =</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="spinX2">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>bw =</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spinBw">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>d0 =</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QDoubleSpinBox" name="spinD0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ra2 =</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="spinRa2">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>rf2 =</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="spinRf2">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>alpha =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="spinAlpha">
          <property name="decimals">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>z0 =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDoubleSpinBox" name="spinZ0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>x0 =</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="spinX0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>ha =</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="spinHa">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>da0 =</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="spinDa0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="spinC">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>c =</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Модификация:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboModification"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>0 точка</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinMod0">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Середина</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinModCenter">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Венец</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinModBw">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Отклонения от теоретического профиля (мкм):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Отклонения относительно максимальной точки (мкм):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
